Applications/spire/source/book_view/technicals_panel.cpp

```diff
@@ -91,3 +132,23 @@ void TechnicalsPanel::set_labeled_data_short_form_text() {
   m_close_label_widget->set_label_text(tr("C"));
   m_volume_label_widget->set_label_text(tr("V"));
 }
+
+void TechnicalsPanel::on_close_signal(const Money& close) {
+  m_close_label_widget->set_data_text(Beam::ToString(close).c_str());
```

Comment:
Change all instances of converting from std::string to QString using s.c_str() to QString::fromStdString(s);

Do it anywhere you might have done this in the past by doing a Search/Find across all files. This avoids issues with performing UTF-8 conversions.

Reply:
Do you want this on *any* call to c_str()? There's some calls to it in the models that don't use any Qt classes.

Reply:
I can look over the other uses of it. Generally speaking c_str() is not advisable but I will briefly check it now and get back to you.